ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/KsqlResourceFunctionalTest.java

```diff
@@ -129,12 +128,14 @@ public void shouldHandleInterDependantExecutableAndNonExecutableStatements() {
   @Test
   public void shouldHandleInterDependantCsasTerminateAndDrop() {
     // When:
+    final int offset = TEST_HARNESS.getKafkaCluster().getAvailableRecords(COMMAND_TOPIC_NAME).size();
```

Comment:
Though probably not possible in this test, this may not be the actual offset that gets uses (kafka will clean up records internally). A more "correct" way of doing this would be to list the queries after the `CREATE STREAM` call and use the id you get back as the id of the query in the terminate call.